ubuntudesign/maas.io
# Asset server update

## Done

- Updates all assets to be using the asset server
- Any images where possible should be now using the responsive srcset attribute

## QA

Pull the branch and run ```make develop``` then view all  pages to make sure all assets are loading via the asset server and the same for all CSS items.

## Issue / Card

#66 

## Screenshots

_n/a_
